plctestbench/node.py

```diff
@@ -117,6 +127,10 @@ def get_data(self) -> np.ndarray:
     def get_original_track_node(self) -> OriginalTrackNode:
         return self.root
     
+    def load(self) -> None:
```

Comment:
Perché tutti questi load?

Reply:
Perché io utilizzo la gerarchia dei nodi del plctestbench per caricare i file audio, in modo da essere sicuro che il risultato sia per definizione lo stesso.

Reply:
Di questo è meglio parlarne a voce e magari guardare il codice insieme

Reply:
I load verranno rimossi. Al loro posto verrà sviluppata una funzione che accetta un run_id ed una lista di node_id per ritornare un plc_testbench con l'intera struttura dei nodi del run. Quelli specificati nella lista contengono il file a loro associato.